docs/src/pages/components/links/Links.js

```diff
@@ -10,21 +11,19 @@ const useStyles = makeStyles(theme => ({
   },
 }));
 
-// This resolves to nothing and doesn't affect browser history
-const dudUrl = 'javascript:;';
```

Comment:
These are [deprecated in 16.9](https://github.com/facebook/react/blob/master/CHANGELOG.md#1690-august-8-2019) and preventing the default works just as well.